### PR TITLE
High contrast

### PR DESCRIPTION
**Pull request by marcomontoya**
Set the colors for the elements of Graphing Calculator in high contrast mode.

### CODE REVIEW

File: src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml [hunk 1]
```diff
@@ -151,37 +151,71 @@
...
-                                          VerticalAlignment="Stretch"
-                                          Background="{TemplateBinding EquationColor}"
-                                          Foreground="{StaticResource SystemChromeWhiteColor}"
-                                          BorderThickness="0">
+                                    x:Uid="equationAnalysisBack"
+                                    MinWidth="44"
+                                    MinHeight="44"
+                                    VerticalAlignment="Stretch"
+                                    BorderThickness="1">
                                 <Button.Content>
```
Comment (rudyhuyn) [inline src/Calculator/Views/GraphingCalculator/KeyGraphFeaturesPanel.xaml]: NIT: <Button.Content> is not necessary ;)

File: src/Calculator/App.xaml [hunk 2]
```diff
@@ -1968,16 +2008,20 @@
...
+                                              Grid.Column="1"
                                               MinWidth="44"
                                               MinHeight="44"
                                               VerticalAlignment="Stretch"
-                                              Background="{TemplateBinding EquationColor}"
-                                              Foreground="{StaticResource SystemChromeWhiteColor}"
-                                              BorderBrush="{TemplateBinding EquationColor}">
+                                              Background="{ThemeResource EquationButtonBackground}"
+                                              Foreground="{ThemeResource EquationButtonForeground}"
+                                              BorderBrush="{ThemeResource EquationBoxBorderBrush}">
```
Comment (rudyhuyn) [inline src/Calculator/App.xaml]: BorderBrush is not consistent with the default style when the button in the following states: Hover (should be green with High Contrast #1 and White when Checked (CheckedHover and CheckedPressed are ok)

File: src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml [hunk 3]
```diff
@@ -389,6 +389,18 @@
...
+                            <ResourceDictionary x:Key="HighContrast">
+                                <SolidColorBrush x:Key="ToggleButtonBackgroundPointerOver" Color="{ThemeResource SystemColorHighlightColor}"/>
+                                <SolidColorBrush x:Key="ToggleButtonForegroundPointerOver" Color="{ThemeResource SystemColorWindowColor}"/>
+                                <SolidColorBrush x:Key="ToggleButtonBackgroundPressed" Color="{ThemeResource SystemColorHighlightColor}"/>
+                                <SolidColorBrush x:Key="ToggleButtonForegroundPressed" Color="{ThemeResource SystemColorWindowColor}"/>
+                            </ResourceDictionary>
+                        </ResourceDictionary.ThemeDictionaries>
+                    </ResourceDictionary>
+                </ToggleButton.Resources>
             </ToggleButton>
```
[the remark this answers is not in the document]
Comment (marcomontoya) [reply src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml]: The default style makes this toggle button behave differently from other toggle buttons in the app.

**Comment by rudyhuyn** [comment]
Taking over this one!

**Pull request closed without merging**